### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (vipulgupta2048) <vipulgupta2048@gmail.com>

Did the best to update the FAQ before the conference. 
@pythonindia/pycon-2018-team Do review the FAQ's and PR to suggest any more changes/additions. 

Are we going to open a form for onsite volunteering, if opened should we update the volunteer page.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -171,14 +170,13 @@ <h5><b>Q. My talk is not selected for conference, can I opt for OpenSpace? </b><
           <ul class="faq-answer">
               <li> Yes. You can submit an OpenSpace proposal once we start accepting the same. Keep following to further updates</li>
           </ul>
-        </div>
+        </div> -->
```
Comment (ananyo2012) [inline faq.html]: Can you uncomment this question as well. Early proposals are accepted at cfp@in.pycon.org

**Comment by vipulgupta2048** [comment]
Gotcha, let me do that in a bit.

**Comment by vipulgupta2048** [comment]
Let's wait for sometime for others to review it as well. If other people would like to add anything.

**Comment by vipulgupta2048** [comment]
Added extra question - https://github.com/pythonindia/inpycon2018/pull/212/commits/16b63a50c93e7ad82fc5d22508e030713f4e4710
All changes implemented.

**Comment by vipulgupta2048** [comment]
Since, we are not having childcare this year, should I remove the page? 
I am reviewing the site for any further changes.

**Comment by ananyo2012** [comment]
Keep it since its just a form link. I will close off the form in some time.
ᐧ

File: faq.html [hunk 2]
```diff
@@ -166,19 +165,18 @@ <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? <
               <li>- PyCon India T-shirt and goodies which will be distributed on Oct 6, 2018.</li>
           </ul>
         </div>
-        <!-- <div class="pycon-faq">
+        <div class="pycon-faq">
           <h5><b>Q. My talk is not selected for conference, can I opt for OpenSpace? </b></h5>
           <ul class="faq-answer">
-              <li> Yes. You can submit an OpenSpace proposal once we start accepting the same. Keep following to further updates</li>
+              <li> Yes. You can submit an OpenSpace proposal once we start accepting the same. Early proposals can be accepted at cfp@in.pycon.org</li>
```
Comment (ananyo2012) [inline faq.html]: link email using mailto

**Comment by vipulgupta2048** [comment]
Changes requested has been implemented.

**Comment by realslimshanky** [comment]
@vipulgupta2048 :taco:

**Pull request merged**